code/modules/projectiles/projectile.dm

```diff
@@ -340,7 +340,7 @@
 /obj/item/projectile/Crossed(atom/movable/AM) //A mob moving on a tile with a projectile is hit by it.
 	..()
 	if(isliving(AM) && AM.density && !checkpass(PASSMOB))
-		Bump(AM, 1)
+		Collide(AM)

```

Comment:
Hmm, this doesn't seem right to me, I'm not sure it should Collide again if it's crossing, as it will already have collided on Enter?